tests/test_pairing_code.py

```diff
@@ -0,0 +1,29 @@
+import unittest
+
+import requests_mock
+
+from canvasapi import Canvas
+from canvasapi.pairing_code import PairingCode
+from tests import settings
+from tests.util import register_uris
+
+
+@requests_mock.Mocker()
+class TestProgress(unittest.TestCase):
```

Comment:
Just noticed that this isn't the right name (should be `TestPairingCode`, right?).